### PR TITLE
#7: add more rules to default config (closes #7)

### LINKED ISSUE #7: add more rules to default config
some experiments in: https://github.omnilab.our.buildo.io/buildo/labonline/blob/web-1416-test_new_eslint_plugins/web/.eslintrc

that `.eslintrc` should contain exactly the proposed diff wrt current `buildo/default`

(note that `no-this-before-super` is already here but disabled (for no reason?))

Some of the rules that I added are a bit opinionated, e.g. `no-plusplus`.. but it made me discover this beauty: https://github.omnilab.our.buildo.io/buildo/labonline/blob/web-1416-test_new_eslint_plugins/web/src/app/components/Basic/Tabs/Tabs.js#L43-L56

Let's discuss these and/or other possible additions in this issue

### PR DESCRIPTION
**Pull request by giogonzo**
Issue #7

### CODE REVIEW

**Comment by giogonzo** [comment]
@FrancescoCioria @gabro should we remove comments for each specific rule completely? they are easy to lookup from eslint web site and mostly self-explanatory, looks like duplication to me

**Comment by FrancescoCioria** [comment]
For me it's ok to remove them

File: rules/default.js [hunk 1]
```diff
@@ -15,42 +15,25 @@ module.exports = {
...
+    "constructor-super": 2,
+    "no-const-assign": 2,
+    "no-this-before-super": 2,
+    "no-use-before-define": 2,
+    "no-var": 2,
+    "prefer-const": 2,
+    "no-class-assign": 2,
+    "no-cond-assign": 2,
+    "no-new": 2,
+    "no-param-reassign": [2, {"props": false}],
```
Comment (gabro) [inline rules/default.js]: props is false by default

Comment (giogonzo) [inline rules/default.js]: right, will remove it

I actually spent some time considering `"props": true` but in the end I left it `false`. In LOL would be very easy to fix (2 errors), not sure about other projects.. what do you think?

Comment (giogonzo) [inline rules/default.js]: anyway just removed it for now, we can consider to add it in the future

**Comment by giogonzo** [comment]
@FrancescoCioria looks like everybody is ok with this :)

File: rules/default.js (hunk 1 above)
Comment (gabro) [inline rules/default.js]: param reassign is bad. I'd add it as soon as possible with `props: false`

Comment (giogonzo) [inline rules/default.js]: it's still there I just removed the useless `props: false`

Comment (gabro) [inline rules/default.js]: oh cool

**Pull request merged**